Source/WebCore/page/Quirks.h

```diff
@@ -39,6 +39,7 @@ class HTMLVideoElement;
 class LayoutUnit;
 class PlatformMouseEvent;
 struct SecurityOriginData;
+class WeakPtrImplWithEventTargetData;
```

Comment:
I believe we sort alphabetically for all of `class` then all of `struct`.

Reply:
Ditto.